packages/source-graphql/package.json

```
@@ -0,0 +1,26 @@
+{
+  "version": "1.0.0",
```

```suggestion
  "version": "0.0.0",
```
We'll probably switch to `1.0.0` when Gridsome reaches v1.0 :)